pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -456,6 +452,8 @@ private static void start() {
         get(API.INVITATION, new GetInvitationRoute(), jsonSerializer);
         patch(API.INVITATION, new UpdateInvitationRoute(), jsonSerializer);
```

Comment:
Should we be preventing exposing invitation codes on URLs?

Reply:
Update URLs in other PR.